packages/react-scripts/config/webpack.config.prod.js

```diff
@@ -149,7 +154,7 @@ module.exports = {
         // @remove-on-eject-begin
         query: {
           babelrc: false,
-          presets: [require.resolve('babel-preset-react-app')],
+          presets: [[require.resolve('babel-preset-react-app'), {isUsingLodash: isUsingLodash}]],
```

Comment:
I would like to write only `{ isUsingLodash }` but I'm not sure of which node versions are supported by `react-create-app`